doc/website/blog/2023-02-23-simple-cms-with-ent.mdx

```diff
@@ -0,0 +1,777 @@
+  tasks for reading and writing from the database.
+* **Bundled with Atlas** - Ent is built with a rich integration with [Atlas](https://atlasgo.io), a robust schema 
+  management tool with many advanced capabilities.  Atlas can automatically plan schema migrations for you as
+  well as verify them in CI or deploy them to production for you. (Full disclosure: Ariel and I are the creators and maintainers)
+
+This tutorial is intended for Ent and Go novices who want to start by building a simple project: a very simple content management system.
+
+#### Prerequisities
+* [Install Go](https://go.dev/doc/install)
+* [Install Docker](https://docs.docker.com/get-docker/)
+
+:::info Supporting repo
+
+You can find of the code shown in this tutorial in [this repo](https://github.com/rotemtam/ent-blog-example)
+
+:::
+
+### Step 1: Setting up the database schema
+
+You can find the code described in this step in [this commit](https://github.com/rotemtam/ent-blog-example/commit/d4e4916231f05aa9a4b9ce93e75afdb72ab25799).
+
+Let's start by initializing our project using `go mod init`:
+```
+go mod init github.com/rotemtam/ent-blog-example
+```
+
+Go confirms our new module was created:
+```
+go: creating new go.mod: module github.com/rotemtam/ent-blog-example
+```
+
+The first thing we will handle in our demo project will be to setup our database. We create our application data modelu using Ent. Let's fetch it using `go get`:
+
+```
+go get -u entgo.io/ent@master
+```
+
+Once installed, we can use the Ent CLI to Initialize the models for the two types entities we will be dealing with in this tutorial, the `User` and the `Post`.
+```
+go run -mod=mod entgo.io/ent/cmd/ent new User Post
+```
+
+Notice that a few files are created:
+
+```
+.
+`-- ent
+    |-- generate.go
+    `-- schema
+        |-- post.go
+        `-- user.go
+
+2 directories, 3 files
+```
+
+Ent created the basic structure for our project:
+* `generate.go` - we will see in a bit how this file is used to invoke Ent's code-generation engine.
+* The `schema` directory, with a bare `ent.Schema` for each of the entities we requested.
+
+Let's continue by defining the schema for our entities. This is the schema definition for `User`:
+```go
+// Fields of the User.  
+func (User) Fields() []ent.Field {  
+   return []ent.Field{  
+      field.String("name"),  
+      field.String("email").Unique(),  
+      field.Time("created_at").Default(time.Now()),  
+   }  
+}  
+  
+// Edges of the User.  
+func (User) Edges() []ent.Edge {  
+   return []ent.Edge{  
+      edge.To("posts", Post.Type),  
+   }  
+}
+```
+
+Observe that we defined 3 fields, `name`, `email` and `created_at` (which takes the default value of `time.Now()`).  Since we expect emails to be unique in our system we added that constraint on the `email` field.  In addition, we defined an edge named `posts` to the `Post` type.
```

Comment:
```suggestion
Observe that we defined three fields: `name`, `email` and `created_at` (which takes the default value of `time.Now()`).  Since we expect emails to be unique in our system, we added that constraint on the `email` field.  In addition, we defined an edge named `posts` to the `Post` type.
```

Reply:
Maybe also give a short brief about what an edge is?